app.rb

```diff
@@ -0,0 +1,136 @@
+require './student'
+require './teacher'
+require './book'
+require './rental'
+
```

Comment:
- Kindly In order to include classes, modules, etc in other files you have to use require_relative or require, and when it is some dependency on locating in gem kindly use require `ex: require 'date'.` so here above  I suggest you use require_relative instead of require  `eg: require_relative  './book'`   it will make your code looks more professional   :+1: :+1: :+1: